examples/scan/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
+VOLUME /golem/input
```

Comment:
Do we need the VOLUME and/or WORKDIR?
If not, maybe it's better to remove them?

Or even better: remove this file at all, and write something like in the `market_strategy` example:
```
#   Image based on pure python:3.8-alpine
IMAGE_HASH = "5c385688be6ed4e339a43d8a68bfb674d60951b4970448ba20d1934d"
```